src/utils/config_utils/src/config_utils.c

```diff
@@ -190,11 +205,11 @@ static void ADUC_AgentInfo_Free(ADUC_AgentInfo* agent)
  * @param agentCount Count of objects in agents.
  * @param agents Array of ADUC_AgentInfo objects to free.
  */
-static void ADUC_AgentInfoArray_Free(size_t agentCount, ADUC_AgentInfo* agents)
+static void ADUC_AgentInfoArray_Free(unsigned int agentCount, ADUC_AgentInfo* agents)
```

Comment:
can remove these if get rid of unneeded agents array

Reply:
changed.